scanapi/__main__.py

```diff
@@ -32,6 +31,13 @@ def main():
     type=click.Path(),
     help="Report output path.",
 )
+@click.option(
+    "-n",
```

Comment:
I think that `-nr` would be nice. What do you think?

Reply:
I agree, will change it :)

Reply:
Done! :)

Reply:
nice @JRubics its great.

Reply:
Thanks 🎉